libglusterfs/src/event-epoll.c

```diff
@@ -802,7 +802,7 @@ event_dispatch_epoll(struct event_pool *event_pool)
             ev_data->event_index = i + 1;
 
             ret = gf_thread_create(&t_id, NULL, event_dispatch_epoll_worker,
-                                   ev_data, "epoll%03hx", i & 0x3ff);
```

Comment:
IMO, this was done to handle a coverity warning which complained about length of the string as it was taking only 8 characters. By making it `%d`, coverity complains on string length as at max, it can become 5+9 characters.

Reply:
Why 8? All thread names are always prefixed with `GF_THREAD_NAME_PREFIX`, which is `glfs_`, and finally clamped down to `GF_THREAD_NAME_LIMIT` (which is 16 and imposed by OS API limit) bytes. So, taking trailing zero byte into account, we have 16 - 1 - strlen("glfs_") = 10 bytes for the thread name. For the long name patterns like `rpcrqhnd%d` this makes the room for 99 threads. If we need more, it may be reasonable to use `gf_` as thread name prefix, which makes the room for 9999 threads :-).